success.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
+      h2 {
+        margin: 30px;
+      }
+      button {
+        width: 120px;
+        height: 35px;
+        color: white;
+        position: relative;
+        cursor: pointer;
+        border-radius: 7px;
+        background-color: rgb(4, 114, 114);
+        border: none;
+        margin-left: 25px;
+        font-size: large;
+        font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
+      }
+    </style>
+    <title>Success</title>
+  </head>
+  <body>
+    <h2>Thank You! Successfully Submitted</h2>
```

Comment:
```suggestion
    <h2>Thank You! You have Successfully Submitted the form</h2>
```

I recommend this instead.